### PR TITLE
chore(examples): add sass example

### PR DESCRIPTION
**Pull request by mcansh**
Signed-off-by: Logan McAnsh <logan@mcan.sh>

### CODE REVIEW

**Review by sergiodxa — APPROVED** [review]
Looks great!

File: examples/with-sass/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Welcome to Remix!
```
Comment (kentcdodds) [inline examples/with-sass/README.md:1]: Could you update this to use the same README format as in the template?

File: examples/with-sass/app/routes/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          <a
+            target="_blank"
+            href="https://remix.run/docs"
+            rel="noreferrer noopener"
+          >
+            Remix Docs
+          </a>
+        </li>
+      </ul>
+    </div>
```
Comment (kentcdodds) [inline examples/with-sass/app/routes/index.tsx:34]: Could you remove the irrelevant stuff that can be removed (like this bit) and comment the primary relevant bits?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks for this! In addition to the comments here, let's remove the unnecessary `with-` prefix. That's what the Next.js repo does and it's just extra fluff that's not necessary.

File: examples/with-sass/README.md (hunk 1 above)
Comment (mcansh) [reply examples/with-sass/README.md]: yep sorry, made this before we had a template 😅

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Comment by mcansh** [comment]
haha, just realized i accidentally removed the css import from root.tsx 🤦

**Comment by kentcdodds** [comment]
Whoops 😅 that might be *too* minimal 🤣 I didn't notice. Could you get that fixed?

**Comment by mcansh** [comment]
> Whoops 😅 that might be _too_ minimal 🤣 I didn't notice. Could you get that fixed?

🤣 - #1467

**Pull request merged**